tests/e2e/performance/test_pvc_clone_performance.py

```diff
@@ -97,7 +114,7 @@ def test_clone_create_delete_performance(self, interface_type, pvc_size, file_si
         self.pod_obj.fillup_fs(size=file_size_for_io, fio_filename=file_name)
 
         # Wait for fio to finish
-        fio_result = self.pod_obj.get_fio_results()
+        fio_result = self.pod_obj.get_fio_results(timeout=18000)
```

Comment:
What is the typical time to complete fio job? 18000 for timeout seems like a huge value.